test/chrome.test.js

```diff
@@ -0,0 +1,59 @@
+import { getCurrentTabUrl } from '../src/api/chrome'
+
+let sendResponseSpy
+beforeEach(() => {
+  const originalImpl = global.chrome.runtime.onMessage.addListener // prevent infinite-loop
+  jest
+    .spyOn(global.chrome.runtime.onMessage, 'addListener')
+    .mockImplementation(listener => {
+      const mockSpiedListener = jest.fn(listener)
+      originalImpl(mockSpiedListener)
+    })
+  require('../src/background')
```

Comment:
this looks quite hackish. mixing import with require is not encouraged either. if this really needs to be this way, better put a comment explaining why

Reply:
Ok. I'll attach a comment about desired side effect here ;)